plugins/modules/postgresql_script.py

```diff
@@ -0,0 +1,398 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: postgresql_script
+
+short_description: Run PostgreSQL statements from a file
+
+description:
+- Runs arbitrary PostgreSQL statements from a file.
+- The module always reports that the state has changed.
+- Does not run against backup files.
+  Use M(community.postgresql.postgresql_db) with I(state=restore)
+  to run queries on files made by pg_dump/pg_dumpall utilities.
+
+version_added: '2.1.0'
+
+options:
+  positional_args:
+    description:
+    - List of values to substitute variable placeholders within the file content.
+    - When the value is a list, it will be converted to PostgreSQL array.
+    - Mutually exclusive with I(named_args).
+    type: list
+    elements: raw
+  named_args:
+    description:
+    - Dictionary of key-value arguments to substitute
+      variable placeholders within the file content.
+    - When the value is a list, it will be converted to PostgreSQL array.
+    - Mutually exclusive with I(positional_args).
+    type: dict
+  path:
+    description:
+    - Path to a SQL script on the target machine.
+    - To upload dumps or to execute other complex scripts, the preferable way
+      is to use the M(community.postgresql.postgresql_db) module with I(state=restore).
+    type: path
+  session_role:
+    description:
+    - Switch to C(session_role) after connecting. The specified role must
+      be a role that the current C(login_user) is a member of.
+    - Permissions checking for SQL commands is carried out as though
+      the C(session_role) were the one that had logged in originally.
+    type: str
+  db:
+    description:
+    - Name of database to connect to and run queries against.
+    type: str
+    aliases:
+    - login_db
+  encoding:
+    description:
+    - Set the client encoding for the current session (e.g. C(UTF-8)).
+    - The default is the encoding defined by the database.
+    type: str
+  trust_input:
+    description:
+    - If C(no), check whether a value of I(session_role) is potentially dangerous.
+    - It makes sense to use C(no) only when SQL injections
+      via I(session_role) are possible.
+    type: bool
+    default: yes
+  search_path:
+    description:
+    - List of schema names to look in.
```

Comment:
should we add a comment that this overwrites the default search_path for this execution and doesn't prepend or append it?